src/components/withCurrentUserPersonalDetails.tsx

```diff
@@ -37,7 +38,10 @@ export default function <TProps extends WithCurrentUserPersonalDetailsProps, TRe
         const accountID = props.session?.accountID ?? 0;
         const accountPersonalDetails = personalDetails?.[accountID];
         const currentUserPersonalDetails: CurrentUserPersonalDetails = useMemo(
-            () => (accountPersonalDetails ? {...accountPersonalDetails, accountID} : {}),
+            () =>
+                accountPersonalDetails
+                    ? {...accountPersonalDetails, accountID, timezone: accountPersonalDetails.timezone ? DateUtils.parseTimezone(accountPersonalDetails.timezone) : undefined}
+                    : {},
```

Comment:
@suneox Can you please help me understand the reason for this change. I think we should change the timezone to supported format before saving it to Onyx. Is there any possible scenario where `personalDetails` retrieved from storage has unsupported timezone?

Reply:
Thanks @suneox for the prompt reply. I didn't get you.

> user has save the timezone is not supported to BE and user do not anything action to re-update timezone.

Can you please add more details. User saved timezone will be from the list of `TIMEZONE.ts`, right?  Will there be any case where the saved timezone is not supported by BE? Please let me know if I am missing something.

Reply:
I don't think we need to consider the situation as we are still in development phase and the issue was not reported before. So, I think we are safe to remove the change. Please let me know if you disagree.